Sources/UIDeviceExtensions.swift

```diff
@@ -26,6 +26,9 @@ private let DeviceList = [
     /* iPhone 7 */        "iPhone9,1": "iPhone 7", "iPhone9,3": "iPhone 7",
     /* iPhone 7 Plus */   "iPhone9,2": "iPhone 7 Plus", "iPhone9,4": "iPhone 7 Plus",
     /* iPhone SE */       "iPhone8,4": "iPhone SE",
+    /* iPhone 8 */        "iPhone10,1": "iPhone 8", "iPhone10,4": "iPhone 8",
```

Comment:
I have never touched this part of the codebase. Would you mind telling me what the key and value for this mapping semantically represent?

Reply:
The Key is the hardware string that uniquely identifies every iDevice out there. The mapping is done in order for the user to get "iPhone 8" instead of "iPhone10,1" (Which might be confusing).

Reply:
Cool. I am not going to go out and verify the mappings, so I am approving this as is.